packages/zwave-js/src/lib/commandclass/WakeUpCC.ts

```diff
@@ -455,6 +474,16 @@ export class WakeUpCCIntervalCapabilitiesReport extends WakeUpCC {
 		return this._wakeUpIntervalSteps;
 	}
 
+	private _wakeUpOnDemandSupported: boolean;
+	@ccValue({ minVersion: 3 })
```

Comment:
I think this should be an internal value ID

Reply:
If this is internal, how will the client know if a node supports this feature? Even if the driver doesn't allow the client to have any control over whether or not a command will trigger a Wake Up On Demand, it's definitely useful for the user to know if the feature is supported.